src/goto-programs/string_instrumentation.cpp

```diff
@@ -281,7 +283,10 @@ void string_instrumentationt::do_sprintf(
   {
     error().source_location=target->source_location;
     error() << "sprintf expected to have two or more arguments" << eom;
-    throw 0;
+
+    throw invalid_user_input_exceptiont(
+      "sprintf expected to have two or more arguments",
+      "sprintf(char *str, const char *format, ...);");
```

Comment:
Is the `error()` output above now redundant? It certainly seems to be a bad idea to duplicate text?